### DISCUSSION TITLE
Hacking with Mac M2

### DISCUSSION THREAD

**Discussion by aliteraphassane** [utmapp/UTM#6236] [Linux]
### Hacking with Mac M1, M2 and M3

**Salut tout le monde !
J'ai un MacBook Air M2 et j'aimerais savoir si c'est possible d'utiliser aircrack-ng,airodump-ng et airmon-ng pour pouvoir hacker le mot de passe de mon wifi. J'ai essayé d'utiliser les logiciels de virtualisation mais hélas j'ai pas pu accéder au point d'accès. Merci pour votre aide !**

@garamb1 
@wegank 
@js-john

**Comment by wegank** [comment]
Je n'ai aucune idée pourquoi je suis mentionné, mais ce  n'est tout simplement pas mon domaine, désolé ...
